### PR TITLE
fix: add link rendering in article body

### PR DESCRIPTION
**Pull request by nehasri89**
As part of this PR, added link rendering in the body copy. On web, links will be responsive and will take the font styles similar to paragraphs in addition to being a link of color `#069` and being underline.

- **Web**
screen shot 2018-01-15 at 11 45 33
screen shot 2018-01-15 at 11 45 45

- **IOS**

screen shot 2018-01-14 at 19 52 11

- **ANDROID**

screen shot 2018-01-14 at 20 01 01

### CODE REVIEW

File: packages/markup/markup.web.js [hunk 1]
```diff
@@ -44,6 +51,13 @@ const defaultRenderers = {
         caption={attributes.caption.name}
       />
     );
+  },
+  link(key, attributes, renderedChildren) {
+    return (
+      <TextLink key={key} style={styles.link} url={attributes.href}>
+        {renderedChildren}&nbsp;
```
Comment (fourlastor) [inline packages/markup/markup.web.js]: why the non breaking space after the link?

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/f8b2c92e9d866e24

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/a4e2f57bfcae9f64

File: packages/markup/markup.web.js [hunk 2]
```diff
@@ -44,6 +51,13 @@ const defaultRenderers = {
         caption={attributes.caption.name}
       />
     );
+  },
+  link(key, attributes, renderedChildren) {
+    return (
+      <TextLink key={key} style={styles.link} url={attributes.href} onPress={() => null}>
+        {renderedChildren}
```
Comment (aronshamash) [inline packages/markup/markup.web.js]: that looks better. How did you achieve the space without `&nbsp` ?

Comment (nehasri89) [reply packages/markup/markup.web.js]: I checked again, the space was added inside the link, not outside, so I removed it @aronshamash

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/47d4b66282c00358

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/710fc683600dc976

File: packages/markup/markup.web.js [hunk 3]
```diff
@@ -12,6 +13,12 @@ const styles = StyleSheet.create({
     borderBottomWidth: 1,
     paddingTop: 10,
     paddingBottom: 10
+  },
+  link: {
```
Comment (colincclark) [inline packages/markup/markup.web.js]: The link seems to be misaligned. It is not flush with the surrounding paragrahs...is that correct? I cannot find a Zeplin design for this...

Comment (craigbilner) [reply packages/markup/markup.web.js]: @nehasri89 can we get screenshots?

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/ff3961e965d4e61b

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/8372766ceef1dbb2

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/e9244f14e9ec71f6

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/674f07ab1ec10208

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/42ce73e025f2cc9e

File: packages/article/fixtures/full-article.json [hunk 4]
```diff
@@ -50,6 +50,22 @@
...
+              "name": "link",
+              "attributes": {
+                "href":
+                  "https://www.thetimes.co.uk/article/peers-call-for-action-after-liam-allan-trial-blunder-vnzn0sz89"
+              },
+              "children": [
+                {
+                  "name": "text",
+                  "attributes": {
+                    "value": "police disclosure of evidence"
```
Comment (colincclark) [inline packages/article/fixtures/full-article.json]: there is no space between this link and the next piece of text on web

Comment (nehasri89) [reply packages/article/fixtures/full-article.json]: done

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/6f242cc2707520fe

**Pull request merged**